src/scss/colors.scss

```diff
@@ -0,0 +1,25 @@
+//Main colors
+$game-background: #eee8ea;
+$accent-color: #591D2D;
+$accent-color-hover: #73263A;
+
+//Text colors
+$text: #4d4d4d;
+$text-light: #666666;
+$tile-text: #ffffff;
+$title: #1a1a1a;
+$title-hover: #591D2D;
```

Comment:
What do you think of using some sort of BEM for modifiers - e.g. double dash for `--hover` and `--light`?